PowerEditor/installer/nativeLang/portuguese.xml

```diff
@@ -211,8 +223,8 @@ Translation note:
 					<Item id="43050" name="Inverter marcador"/>
 					<Item id="43052" name="Localizar carateres..."/>
 					<Item id="43053" name="Selecionar tudo entre as chavetas correspondidas"/>
-					<Item id="43009" name="Ir para a chaveta correspondida"/>
 					<Item id="43010" name="Localizar anterior"/>
+					<Item id="43009" name="Ir para a chaveta correspondida"/>
```

Comment:
id="43009" should be before id="43010"

Reply:
Done

Reply:
**id=43011** is not aligned with the text!